docs/authentication/personal-access-tokens.md

```diff
@@ -45,3 +73,38 @@ Without an access token, making programmatic requests will result in a 401 resul
 is enabled.
 
 :::
+
+## Additional Resources
+
+- Learn more about how this feature is by DataHub [Authentication Metadata Service](introducing-metadata-service-authentication.md).
+- Check out our [Authorization Policies](../authorization/policies.md) to see what permissions can be programatically used.
+
+### GraphQL
+
+ - Have a look at [Token Management in GraphQL](../api/graphql/token-management.md) to learn how to manage tokens programatically!
+ 
+## FAQ and Troubleshooting
+
+ <!-- Use the following format:
```

Comment:
Let's remove commented out details from the template unless they capture doc-specifc details